src/client/scripts/room.js

```diff
@@ -1,15 +1,37 @@
 "use strict";
-/* global Cards:false Rooms:false Popup: false*/
+/* global Cards:false Rooms:false Popup: false UserMethods:false*/
 /**
  *@purpose To provide the room template with data to display
  **/
 
 Template.room.onCreated(function() {
     Meteor.autorun(function() {
         Meteor.subscribe("cards", Session.get("roomCode"));
+        if (Meteor.user()) {
+            Meteor.call("addUserToRoom", Session.get("author"), Session.get("roomCode"));
+        }
     });
+    if (Session.get("author")) {
+        Meteor.call("addUserToRoom", Session.get("author"), Session.get("roomCode"), function(err, res) {
+            if (err) {
+                return;
+            }
+            if (res) {
+                Session.set("aliasID", res);
+            }
+        });
+    }
 });
 
+Template.room.onDestroyed(function() {
```

Comment:
Fyi, this does NOT get called if the user leaves the site, probably have to attach to the actual sockets and hook on the disconnect
http://stackoverflow.com/questions/10257958/server-cleanup-after-a-client-disconnects
is kinda what you want